bin/api-compare.php

```diff
@@ -31,7 +40,210 @@
             new GetVersionCollectionFromGitRepository(),
             new PickLastMinorVersionFromCollection(),
             new Comparator(
-                new Comparator\BackwardsCompatibility\ClassBased\PropertyRemoved()
```

Comment:
Note: this tree of checks is quite huge, but is pretty much describing what kinds of BC breaks we want to rely upon.

If we want to make it more "friendly", I can move the instantiation to `Roave\ApiCompare\ApiCompare#defaultClassChecks()`, `Roave\ApiCompare\ApiCompare#defaultInterfaceChecks()`, `Roave\ApiCompare\ApiCompare#defaultTraitChecks()`,